sinatra/lesson1/tutorial.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: page
+title: Introduction to Sinatra
+---
+
+## What is Sinatra?
+
+Sinatra is a super simple framework for making websites using Ruby and HTML.
+It's easy to understand, and a great tool to start learning web development
+with!
+
+Before you continue, make sure you'll completed the [HTML tutorial][html-tut],
+[version control tutorial][vc-tut], and the [Ruby tutorial][ruby-tut].
+
+[html-tut]: http://codebar.github.io/tutorials/html/lesson1/tutorial.html
+[vc-tut]: http://codebar.github.io/tutorials/version-control/introduction/tutorial.html
+[ruby-tut]: http://codebar.github.io/tutorials/ruby/lesson1/tutorial.html
+
+## Installing Ruby
+
+Before you begin, you must have Ruby installed on your machine. There are a
+couple of different ways you can get ruby setup.
+
+- Windows installation using [Ruby installer][ruby-installer]
+- Mac & Linux installation using [rbenv][rbenv-install]
+
+[rbenv-install]: https://github.com/sstephenson/rbenv "rbenv"
+[ruby-installer]: http://rubyinstaller.org/
+
+Once you've got Ruby installed, you'll need to install Sinatra. Open the
+terminal / command prompt, and type the following:
+
+```
+gem install sinatra
```

Comment:
Hm, yes, we should be handling deps correctly. Do you think it should be how we introduce installing gems from the start, or should we first use gem, and then introduce the Gemfile in a later lesson?